card/mod/standard/set/type/session.rb

```diff
@@ -27,7 +27,7 @@ def content= val
 end
 
 format :html do
-  def default_core_args args
-    args[:items] = { view: :name }
+  def default_core_args _args
+    voo.items[:view] = :name
```

Comment:
Are you sure `voo.items` is always initialized?

Reply:
it's a method defined in voo_api.rb